MSAL/src/MSALPublicClientApplication.m

```diff
@@ -115,11 +110,33 @@ + (void)load
     MSIDNotifications.webAuthDidReceiveResponseFromBrokerNotificationName = MSALWebAuthDidReceieveResponseFromBroker;
 }
 
+#pragma mark - Properties
+- (BOOL)validateAuthority { return _configuration.validateAuthority; }
+- (void)setValidateAuthority:(BOOL)validateAuthority { _configuration.validateAuthority = validateAuthority; }
+
+- (MSALAuthority *)authority { return self.configuration.authority; }
+- (NSString *)clientId { return self.configuration.clientId; }
+- (MSALRedirectUri *)redirectUri { return self.configuration.verifiedRedirectUri; }
+
+- (NSDictionary<NSString *,NSString *> *)sliceParameters { return self.configuration.sliceConfig.sliceDictionary; }
+- (void)setSliceParameters:(NSDictionary<NSString *,NSString *> *)sliceParameters
+{
+    self.configuration.sliceConfig = [MSALSliceConfig configWithSlice:sliceParameters[@"slice"] dc:sliceParameters[@"dc"]];
```

Comment:
nit: verify sliceParameters is not empty

Reply:
Added logging warning for passing in a wrong slice info here.